docker-compose.yml

```diff
@@ -207,6 +207,10 @@ services:
             ca.mcgill.a11y.image.handler: enable
         volumes:
             - sc-store:/tmp/sc-store
+    people-handler:
+        image: ghcr.io/shared-reality-lab/image-people-handler:unstable
```

Comment:
Name doesn't fit convention.

Reply:
@jeffbl should the name be image-handler-people?